ui/app/AppLayouts/Chat/views/ChatContentView.qml

```diff
@@ -65,6 +66,9 @@ ColumnLayout {
         property bool isUserAdded
 
         function updateIsUserAdded() {
+            if (activeChatType !== Constants.chatType.oneToOne) {
```

Comment:
I wonder how does this comparison ever work :)

Reply:
It didn't, was always hitting the `then` block. :)